src/bpftrace.h

```diff
@@ -130,6 +134,9 @@ class BPFtrace
   static volatile sig_atomic_t exitsig_recv;
 
   std::map<std::string, std::unique_ptr<IMap>> maps_;
+  //
```

Comment:
No i actually wanted to place a comment here, just forgot to actually write it :p. Wanted to avoid adding yet-another-undocumented-variable to bpftrace.h